src/toolkits/object_detection/object_detector.cpp

```diff
@@ -779,6 +797,13 @@ void object_detector::perform_predict(gl_sframe data,
       input_batch[i].annotations.clear();
     }
 
+    // Store image dimentions
+    result_batch.img_dimensions_batch.resize(input_batch.size());
```

Comment:
Instead of storing this state I’m the `result_batch`, let’s create a new struct that contains the result batch and, say, the entire original vector of `labeled_image` instances. This struct can be defined in the header (in the private section of the class)

Reply:
good!